tabpy/tabpy_server/handlers/evaluation_plane_handler.py

```diff
@@ -44,6 +44,11 @@ def initialize(self, executor, app):
 
     @gen.coroutine
     def _post_impl(self):
+        if not self.eval_enabled:
```

Comment:
Sorry, just got a chance to re-visit.
But can we move this check to 
https://github.com/tableau/TabPy/blob/938ce3b0268435c2738f8b6aa0f30fccd2f82a9d/tabpy/tabpy_server/handlers/evaluation_plane_handler.py#L107
So that we can refuse the request a bit more earlier.
And 404 would be more appropriate I think.